service/service.go

```diff
@@ -0,0 +1,58 @@
+package service
+
+import (
+	"encoding/json"
+	"github.com/Jeffail/gabs"
+	"github.com/blockassets/cgminer_client"
+	"io"
+	"io/ioutil"
+	"log"
+	"os/exec"
+	"time"
+)
```

Comment:
Order of imports please.

Reply:
damn interaction between gitfmt and intelij :-(